io.github.jazztickets.irrlamb.yml

```diff
@@ -0,0 +1,24 @@
+app-id: io.github.jazztickets.irrlamb
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: irrlamb
+rename-desktop-file: irrlamb.desktop
+rename-icon: irrlamb
+
+modules:
+  - name: irrlamb
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: https://github.com/jazztickets/irrlamb/releases/download/v1.0.2/irrlamb-1.0.2r648-src.tar.gz
+        sha256: 46c559846bbc73949682905b7c75d3b174366ed39cc57cd8ea68ff7b942a46c9
+      - type: file
+        path: io.github.jazztickets.irrlamb.appdata.xml
+    builddir: true
+
+finish-args:
```

Comment:
finish args usually for above modules

Reply:
Do you mean it's placed above the modules section?